.jshintrc

```diff
@@ -0,0 +1,16 @@
+{
+    "node": true
+  , "bitwise": true
+  , "undef": true
+  , "trailing": true
+  , "quotmark": false
+  , "indent": 4
+  , "unused": "vars"
+  , "latedef": "nofunc"
+  , "jasmine": true
```

Comment:
Setting `jasmine` here seems excess, as there is only one file with tests. Maybe just add inline `/* jshint jasmine: true*/` to `tests.js`